alacritty_terminal/src/display.rs

```diff
@@ -368,7 +368,10 @@ impl Display {
             }
         }
 
-        if font_changed || self.last_message != terminal.message_buffer_mut().message() {
+        // Message bar message update detected
```

Comment:
I think `Message bar update detected` would be a little easier to read here.

```suggestion
        // Message bar update detected
```